modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectFieldAPI.macro

```diff
@@ -86,6 +86,27 @@ definition {
 		return ${staticObjectFieldId};
 	}
 
+	macro updateAllFieldsOfEntryById {
```

Comment:
See https://github.com/liferay-headless/liferay-portal/pull/1229/files#r1190727008. And I think this should be in CustomObjectAPI.macro file

Reply:
@magjed4289 what do you think of it?